frontend/src/core/comments/components/AddComment.js

```diff
@@ -25,69 +34,87 @@ export default function AddComments(props: Props) {
         addComment,
     } = props;
 
-    const commentInput: any = React.useRef();
-    const minRows = 1;
-    const maxRows = 6;
+    let [editorState, setEditorState] = React.useState(
+        EditorState.createEmpty()
+    )
+    const editor: any = React.useRef(null);
+    const rawContentState = convertToRaw(editorState.getCurrentContent());
 
-    if (!user) {
-        return null;
+    const onChange = (editorState) => {
+        setEditorState(editorState);
     }
 
-    const handleOnChange = () => {
-        const textAreaLineHeight = 24;
-        commentInput.current.rows = minRows;
-
-        const currentRows = Math.trunc(commentInput.current.scrollHeight / textAreaLineHeight);
-
-        if (currentRows < maxRows) {
-            commentInput.current.rows = currentRows;
-        }
-        else {
-            commentInput.current.rows = maxRows;
-        }
+    function focusEditor() {
+        editor.current.focus();
     }
 
-    const handleOnKeyDown = (event: SyntheticKeyboardEvent<>) => {
-        if (event.keyCode === 13 && event.shiftKey === false) {
-            submitComment(event);
+    function keyBindingFn(e: SyntheticKeyboardEvent<>) {
+        if (e.keyCode === 13 && e.shiftKey === false) {
+          return 'submitOnEnter' 
         }
+        
+        return getDefaultKeyBinding(e)
     }
 
-    const submitComment = (event: SyntheticEvent<>) => {
-        event.preventDefault();
-        const comment = commentInput.current.value;
+    React.useEffect(() => {
+        focusEditor()
+    }, []);
 
-        if (!comment) {
-            return null;
```

Comment:
Looks like we lost that - empty comments can be submitted.

Reply:
This check is no longer working.  I will continue to work on this to find out how to prevent an empty entry.

Reply:
Found the fix for this.

However, in the process I discovered a bug that only occurs in the translation comments but happens on both the original comment implementation and with this editor version. 
>When you submit an empty comment, then submit a normal comment, and then try to submit an empty comment again the `Same translation already exists` error displays.

Reply:
Thanks! The bug happens because focus is set to the text editor after you successfully submit a comment. It shouldn't be related to preventing empty comments from being submitted.

If it's a non-trivial fix, we can add it to the list of bugs to be filed and fixed in a followup. I usually keep such list in the first comment of the PR and file bugs after merging.